cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,74 @@
 /// <reference types='cypress' />
 
+const faker = require('faker');
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should have a button to switch pages', () => {
+    cy.contains('.-btn', 'Next').should('exist');
+  });
+
+  it('should be able to select row count', () => {
+    cy.get('select').select('25 rows');
+    cy.get('select').should('contain', '25 rows');
+  });
+
+  it('should be add a New Yorker.', () => {
+    const firstName = faker.name.firstName();
+    const lastName = faker.name.lastName();
+    const email = faker.internet.exampleEmail(firstName, lastName);
+    const age = faker.random.number({ min: 25, max: 60 });
+    const salary = faker.random.number({ min: 2000, max: 6000 });
+    const department = faker.random.number({ min: 1, max: 3 });
+
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(firstName);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(email);
+    cy.get('#age').type(age);
+    cy.get('#salary').type(salary);
+    cy.get('#department').type(department);
+    cy.get('#submit').click();
+    cy.get('.rt-tr')
+      .should('contain', firstName)
+      .and('contain', lastName);
+  });
+
+  it('should be delete a worker', () => {
```

Comment:
same here - it should be able to delete a worker. Also make changes in namings everywhere, please)